android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.kt

```diff
@@ -127,4 +134,25 @@ class FullScreenPlayerView(
         layoutParams.setMargins(0, 0, 0, 0)
         return layoutParams
     }
+
+    private fun updateNavigationBarVisibility() {
```

Comment:
reviewing again the code, I wonder why did you choose the manage vibility here, where there is a similar code in: 
`public void setFullscreen(boolean fullscreen) {`
I am afraid we will be lost if we split 'fullscreen' responsibility...

Reply:
In the `setFullScreen` function, we handle showing the navigation bar and status bar on the first screen, but now we want to handle it in Dialog (full-screen) mode. I think this is the correct way to handle it in this class `(FullScreenPlayerView)`.